tests/20-basic-component-check.py

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/python3
 
 import amulet
-import os
 import unittest
-import yaml
 
-from amulet_utils import attach_resource
 from amulet_utils import check_systemd_service
 from amulet_utils import kubectl
 from amulet_utils import run
 from amulet_utils import valid_certificate
 from amulet_utils import valid_key
 
-SECONDS_TO_WAIT = 1200
+# Turn this down. With reset=false in tests.yaml we dont have to wait for
+# standup
+SECONDS_TO_WAIT = 300
 
 
 class IntegrationTest(unittest.TestCase):
```

Comment:
The docstring is no longer valid and if you are in here we should update that to reflect what is going on with this test.